dali/dfu/dfurun.cpp

```diff
@@ -1317,7 +1317,7 @@ class CDFUengine: public CInterface, implements IDFUengine
 #ifdef _CONTAINERIZED
                         StringBuffer clusterName;
                         destination->getGroupName(0, clusterName);
-                        Owned<IPropertyTree> plane = getDropZonePlane(clusterName);
+                        Owned<IPropertyTree> plane =  getStoragePlane(clusterName);
```

Comment:
This is a bug fix.  The original code should have used getStoragePlane() not getDropzonePlane.